crt_portal/cts_forms/templates/forms/snippets/file_upload.html

```diff
@@ -0,0 +1,33 @@
+<table class="usa-table usa-table--borderless attachments-table width-full">
+  <tbody id="attachmentTable">
+    {% if pro_form_attachments %}
+        {% for attachment in pro_form_attachments %}
```

Comment:
I think this `if` is unnecessary; the for loop will simply be empty if pro_form_attachments is None or empty